### ISSUE TITLE
React-Native Link (i.e. no cocoa pods)

### ISSUE THREAD

**Issue by dorthwein** [rastapasta/react-native-gl-model-view#2]
Any chance on making either a react-native link or a work around to not have to use cocoa pods?

**Comment by dorthwein** [comment]
...and I need more coffee....sorry need to read more hahah

**Comment by rastapasta** [comment]
@dorthwein you could just manually add the GLView package:

> To use GLView, just drag the class files into your project and add the QuartzCore and OpenGLES frameworks. If you are using the optional GZIP library then you will also need to add the libz.dylib. 

=> [GLView repository](https://github.com/nicklockwood/GLView#installation)

**Comment by dorthwein** [comment]
@rastapasta - I went ahead and forked the Repo and added the GLView files directly into the xcode package, and configured them so the app will build etc...

The end result is that you can now simply do an `npm install react-native-gl-model-view` and then `react-native link react-native-gl-model-view` to have the package set up & ready to go!

I need to up date the docs for this functionality though.  Would you like me to do a pull request to master?

Also I'm not sure if any of the changes would have broken Cocoa Pods.

**Comment by HofmannZ** [comment]
@dorthwein Any chance we'll we'll a PR on this repo?

**Comment by RadiksMan** [comment]
@dorthwein , please tell how to make work without cocoa pods

**Comment by dorthwein** [comment]
sorry for the radio silence on this.  I actually don't remember what we did on this and ended up not using it as a result of no android support (even offered to pay @rastapasta to port it). 

Anyway @rastapasta suggestion above is the way to go in that you just need to add the files to your project.  

(1) Manually Add the xcode poject to your Libraries folder by right clicking it and select "Add files to *project_name*
(2) node_modules/react-native-gl-model-view/ios/RNGLModelView.xcodeproj
(3) Drag all the .h/.m files into your *project_name* file
(4) Go to Build Phase -> Link Binary with Libraries -> QuartzCore and OpenGLES 
(5) give it a shot?  

This is really rough memory pull and likely not everything - you'll have to debug through it a bit.  We did get it going at some point over the summer and it worked pretty well!
